packages/modules/src/modules/banners/momentTemplate/components/MomentTemplateBannerVisual.tsx

```diff
@@ -79,6 +79,10 @@ const container = (bannerId?: BannerId) => css`
 
     ${from.tablet} {
         height: 100%;
-        width: 100%;
+        width: ${bannerId === 'global-new-year-banner' ? '110%' : '100%'};
+    }
+    ${from.wide} {
+        height: 100%;
+        width: ${bannerId === 'global-new-year-banner' ? '135%' : '100%'};
     }
```

Comment:
I really think we should avoid having overrides like this in the template as we risk having all kinds of legacy conditional overrides in the code. If the design doesn't match the template then it's something we should push back-on, if it's a design feature that'll happen again, then we should make the template extensible in a more generic fashion. This could be added to the issues @marialani is collating for improvements to the template.

Reply:
We do unfortunately have a few of these in atm but do want to get rid of them. We caught up on improvements to the moment template in the SR dev chat and we've a spike doc Rik is leading on  [here](https://docs.google.com/document/d/16hGHFvdXafQkMfhsfuQH99d3Ui1GQjlLwh0aFcPe4CE/edit#heading=h.opr080actx13), will make sure this is on there with an example

Reply:
From what I can tell the reason we've got these values in here is so as to 'zoom' the picture in a little on those breakpoints. I might suggest asking for different images for these breakpoints rather than adding in these conditional overrides

Reply:
Yes, will do Maria/George, I think as its a zoom I can do this in the image itself so no need to resize here thus making it easier for Rik to automate later.

Reply:
Ok, after much tinkering with image uploads in the Grid have come to the conclusion that the width/height of the (Figma) image supplied (especially at 1300px+) is wider than it is tall (rectangular). The image needs to be changed into a squarer shape  on the template banner (aspect ratio same) or the width/height needs expanding via this code to encompass the rectangle. Will contact Jade, try to explain we don't want this exception, and check adjustments I have done are acceptable.